traffic_stops/static/js/app/states/md/defaults.js

```diff
@@ -0,0 +1,48 @@
+import d3 from 'd3';
+
+// Traffic Stops global defaults
+export default {
+  start_year: 2012,     // start-year for reporting requirement
+  end_year: new Date().getUTCFullYear(),       // end-date for latest dataset
+  ethnicities: [
+    'WHITE',
+    'BLACK',
+    'NATIVE AMERICAN',
+    'HISPANIC',
+    'ASIAN',
+    'OTHER',
+    'UNKNOWN',
```

Comment:
As noted in the backend code, the MD import doesn't keep both OTHER and UNKNOWN ethnicities.